docs/notebooks/ppg/ppg_analysis.ipynb

```diff
@@ -99,12 +105,27 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "#TODO: Not fully implemented\n",
+    "# config = HeartRateClassificationConfig()\n",
+    "# signal_quality_classification(path_to_quality_features, quality_classifier_path, path_to_signal_quality, config)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#TODO: Not implemented\n",
```

Comment:
Deze werkt wel (een beetje)? Of moet dit ook uitgecommend worden?

Reply:
uitgecommend idd